sphinx/builders/html.py

```diff
@@ -87,6 +89,35 @@ def get_stable_hash(obj):
     return md5(text_type(obj).encode('utf8')).hexdigest()
 
 
+class CSSContainer(list):
+    """The container of stylesheets.
+
+    To support the extensions which access the container directly, this wraps
+    the entry with Stylesheet class.
+    """
+    def append(self, obj):
+        warnings.warn('builder.css_files is deprecated. '
```

Comment:
`app.add_stylesheet` calls `StandaloneHTMLBuilder.css_files.append(css)`, so this warning will be emitted always (normally it is not displayed now because RemoveInSphinx20Warning) even if user used `app.add_stylesheet`.
Is it intended? or I misunderstand this implementation?